tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
-allowlist_externals=./check_woke.sh
-skipsdist=True
-envlist = lint, woke, unit, functional
+allowlist_externals = ./check_woke.sh
+skipsdist = True
+envlist = lint, unit, integration
```

Comment:
The `integration` target will be run also in the `integration_test`workflow. The targets defined here are run as part of the unit test step. Hence, I recommend you remove the `integration`target from this list

Reply:
Ah thanks, will update.